src/main/java/org/spongepowered/api/item/inventory/property/AbstractInventoryProperty.java

```diff
@@ -93,7 +94,7 @@ protected AbstractInventoryProperty(@Nullable K key, @Nullable V value) {
         this(key, value, null);
     }
 
-    protected AbstractInventoryProperty(@Nullable K key, @Nullable V value, Operator op) {
+    protected AbstractInventoryProperty(@Nullable K key, @Nullable V value, @Nullable Operator op) {
```

Comment:
Actually never mind, these are the protected constructors so it totally is nullable, ignore me.